content/references/rippled-api/admin-rippled-methods/logging-and-data-management-methods/download_shard.md

```diff
@@ -45,15 +45,23 @@ An example of the request format:
 }
 ```
 
+*Commandline*
+
+```sh
+# Syntax: download_shard [[<index> <url>]]
+rippled download_shard 1 https://example.com/1.tar.lz4 2 https://example.com/2.tar.lz4 5 https://example.com/5.tar.lz4
+```
+
 <!-- MULTICODE_BLOCK_END -->
 
 
-The request includes the following fields:
+The request includes the following field:
 
 | `Field`    | Type    | Description                                           |
 |:-----------|:--------|:------------------------------------------------------|
 | `shards`   | Array   | List of Shard Descriptor objects (see below) describing shards to download and where to download them from. |
-| `validate` | Boolean | _(Optional)_ If `false`, skip validating the downloaded data. The default is `true`, which checks that the shard in the archive contains all the data objects for the shard and the shard is part of the ledger history of the current validated ledger. |
+
+The `validate` field is deprecated and may be removed in a future version. (The server always checks the integrity of shards when it imports them.) [Updated in: rippled 1.6.0][]
```

Comment:
Not super familiar with markdown, why the empty square brackets at the end?

Reply:
It's a link that's defined elsewhere. (The [actual file with the definitions](https://raw.githubusercontent.com/ripple/xrpl-dev-portal/v1.6.0/content/_snippets/rippled_versions.md) is imported into this one using some non-standard syntax.)